### PR TITLE
fix: use of Heurist model env vars

### PR DESCRIPTION
**Pull request by boxhock**
# Relates to:

# Risks

Low: If the user had previously set these env vars to any value, these will now be used where previously they would be ignored.

# Background

## What does this PR do?

This PR will make the previously defined Heurist model env vars actually take effect.

## What kind of change is this?

Bug fix

## Why are we doing this? Any context or related work?

Because the env vars are already defined, and it makes sense to use them.

# Documentation changes needed?

My changes do not require a change to the project documentation.

# Testing

## Where should a reviewer start?

The change should be pretty straight forward.

### CODE REVIEW

**Review by monilpat — APPROVED** [review]
LGTM! :)

File: packages/core/src/models.ts [hunk 1]
```diff
@@ -289,11 +289,11 @@ export const models: Models = {
         },
         endpoint: "https://llm-gateway.heurist.xyz",
         model: {
-            [ModelClass.SMALL]: "meta-llama/llama-3-70b-instruct",
-            [ModelClass.MEDIUM]: "meta-llama/llama-3-70b-instruct",
-            [ModelClass.LARGE]: "meta-llama/llama-3.1-405b-instruct",
+            [ModelClass.SMALL]: settings.SMALL_HEURIST_LANGUAGE_MODEL || "meta-llama/llama-3-70b-instruct",
```
Comment (oxSaturn) [inline packages/core/src/models.ts:292]:
```suggestion
            [ModelClass.SMALL]: settings.SMALL_HEURIST_MODEL || "meta-llama/llama-3-70b-instruct",
```
Probably update it to maintain consistency with the others?
https://github.com/ai16z/eliza/blob/main/.env.example#L73-L88

Comment (wjw12) [reply packages/core/src/models.ts:292]: The update sgtm!

Comment (tsubasakong) [reply packages/core/src/models.ts:292]: nice!

**Review by shakkernerd — DISMISSED** [review]
Nice addition.

**Review by shakkernerd — APPROVED** [review]

**Review by shakkernerd — APPROVED** [review]

**Pull request merged**